doc/colvars-refman-main.tex

```diff
@@ -958,13 +958,22 @@
 
 \cvsubsec{Input state file}{sec:colvars_input}
 
-Several of the sampling methods implemented in Colvars are time- or history-dependent, i.e.\ they work by accumulating data as a simulation progresses and use this data to change their biasing forces.  When continuing a simulation over consecutive runs, a \emph{state file} needs to be loaded into Colvars.
+Several of the sampling methods implemented in Colvars are time- or history-dependent, i.e.\ they work by accumulating data as a simulation progresses and use these data to determine their biasing forces.
+
+This requires that the Colvars information 
+
+a standard method for checkpointing (currently, these are only GROMACS and LAMMPS)
```

Comment:
Part of the sentence is missing, and I'm unsure what the intention was exactly.

Reply:
Oops, working on multiple windows. My employer may not be ready yet to shut down, but my brain may be :-)